packages/core/strapi/tests/api/populate/filtering/index.test.e2e.js

```diff
@@ -300,5 +300,19 @@ describe('Populate filters', () => {
 
       expect(body.data[0].attributes.third.data[0].attributes.fooRef).toBeUndefined();
     });
+
+    test("Don't populate with object and 'f'", async () => {
+      const qs = {
+        populate: {
+          third: 'f',
```

Comment:
I'm not sure what I think about this. IMO it doesn't make sense to make this a possibility as our populate algorithm only works by picking what you want not omitting what you don't want 🤔 

It would make sense if we could say populate everything but `third` but that's not the case.

Reply:
I understand. I see it as "If I can put `true` than I can also put `false`". It's weird to put false and to see it populated.
I also imagine it of how it can be used programmatically: `shouldPopulate ? true : false` (or just `shouldPopulate`).